### PR TITLE
User console

### PR DESCRIPTION
**Pull request by Koorae**
Note: Please reference the issue ids that this merge request resolves. If your description add "Closes #issue-id" or "Fixes #issue-id", GitHub will close the issues referenced. 

## Summary:
Adds a page with the contributions from a user (completes issue #80), and includes several finalizing bug fixes, documentation, and database seed updates

## Expectation:
Resolves issue #80

## Tests:
Tested together and individually by @mikekben  @PleaseAndThankYou

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,8 +17,6 @@ gem 'turbolinks',                 '5.2.1'
 gem 'jbuilder',                   '2.10.0'
 gem 'bootsnap',                   '1.4.6', require: false
 gem 'devise'
-gem 'sendgrid-ruby'
-gem 'figaro'
```
Comment (mikekben) [inline Gemfile:21]: Gems were originally considered for use with email verification, but weren't used in the end

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -179,9 +179,9 @@ def filter
   
   def admin_filter
     if session[:filter]!=nil and not session[:filter].blank? and not session[:filter]=="All"
-      @approveds = @approveds.where(category: session[:filter])
-      @pendings = @pendings.where(category: session[:filter])
-      @denieds = @denieds.where(category: session[:filter])
+      @approveds = @approveds.where(category: session[:filter].downcase)
+      @pendings = @pendings.where(category: session[:filter].downcase)
+      @denieds = @denieds.where(category: session[:filter].downcase)
```
Comment (mikekben) [inline app/controllers/items_controller.rb:184]: Evidence for why categories need to be revamped (see issue #81)

File: app/views/items/admin_index.html.erb [hunk 3]
```diff
@@ -30,7 +30,7 @@
         <label class="form-check-label">Audio Books</label>
       </div>
       <div class="form-check form-check-inline" style="padding-top: 0;">
-        <input class="form-check-input" type="radio" name="filter" id="inlineRadio2" value="Website" onclick="this.form.submit()" <% if session[:filter]=="Website" %>checked<% end%>>
+        <input class="form-check-input" type="radio" name="filter" id="inlineRadio2" value="Online" onclick="this.form.submit()" <% if session[:filter]=="Online" %>checked<% end%>>
```
Comment (mikekben) [inline app/views/items/admin_index.html.erb:33]: See issue #81

**Review by mikekben — COMMENTED** [review]
Everything looks good and matches testing agreed to in meeting

**Review by practiceacc-eng — APPROVED** [review]

**Pull request merged**